requirements.txt

```diff
@@ -40,7 +40,7 @@ sentry_sdk>=1.6,<2
 siphashc>=1.2,<3.0
 social-auth-app-django==5.0.0
 social-auth-core==4.3.0
-translate-toolkit>=3.8.3,<3.9
+translate-toolkit>=3.8.6,<3.9
```

Comment:
The dependency bump is not really needed, the format will just not be supported if using an older version.

Reply:
Ohh that's cool ! Thanks